pallas.csl

```diff
@@ -0,0 +1,282 @@
+    <title-short>Pallas</title-short>
+    <id>http://www.zotero.org/styles/pallas</id>
+    <link href="http://www.zotero.org/styles/pallas" rel="self"/>
+    <link href="https://journals.openedition.org/pallas/2624" rel="documentation"/>
+    <author>
+      <name>Louis Autin (SU)</name>
+      <email>louis.autin@sorbonne-universite.fr</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="humanities"/>
+    <issn>0031-0387</issn>
+    <eissn>2272-7639</eissn>
+    <summary>Zotero stylesheet for Pallas (journal)</summary>
+    <published>2022-09-02T00:00:00</published>
+    <updated>2022-09-02T00:00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <!-- INSTRUCTIONS DE FORMATAGE EN FRANCAIS -->
+  <locale>
+    <terms>
+      <term name="editor" form="short">
+        <single>éd.</single>
+        <multiple>éd.</multiple>
+      </term>
+      <term name="translator" form="verb-short">trad.</term>
+      <term name="in">dans</term>
+      <term name="no date">pas de date</term>
+    </terms>
+  </locale>
+  <!-- MACROS DE CITATION -->
+  <macro name="auteurs-court">
+    <names variable="author">
+      <name form="short" initialize-with=". " and="text" delimiter=", "/>
+      <et-al font-style="italic"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <text macro="titre"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="indication-localisation">
+    <group delimiter=", ">
+      <choose>
+        <if type="book report thesis" match="any">
+          <group>
+            <text term="volume" form="short" suffix=" "/>
+            <number variable="volume" form="numeric"/>
+          </group>
+        </if>
+      </choose>
+      <group delimiter=" ">
+        <choose>
+          <if locator="page" match="none">
+            <label variable="locator" form="short" suffix=" "/>
+          </if>
+        </choose>
+        <text variable="locator" prefix="p. "/>
+      </group>
+    </group>
+  </macro>
+  <!-- MACROS DE BIBLIOGRAPHIE -->
+  <macro name="contributeurs-tri">
+    <names variable="author">
+      <name name-as-sort-order="all" and="text" sort-separator=" " delimiter=", " delimiter-precedes-last="never"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="auteurs">
+    <names variable="author">
+      <name name-as-sort-order="all" and="text" sort-separator=", " delimiter=", " initialize-with=". " font-variant="small-caps" delimiter-precedes-last="never"/>
+      <label form="short" prefix=" (" suffix=")"/>
+      <substitute>
+        <names variable="editor" font-variant="small-caps"/>
+        <names variable="translator" font-variant="small-caps"/>
+      </substitute>
```

Comment:
you almost certainly want to add title here to correspond to the in-text citation (also look at the weird webpage example in the preview)